lib/ansible/modules/packaging/os/apt_key.py

```diff
@@ -32,7 +32,7 @@
         required: false
```

Comment:
Not your cause, but we tend to remove the following from parameter docs:
```
  required: false
  default: none
```
As they are implicit.